src/app/search-page/search-page.component.ts

```diff
@@ -135,6 +140,15 @@ export class SearchPage implements OnInit, OnDestroy {
     })
   }
 
+  private injectAdvancedSearchSurvey(): void{
+    const scriptSrc = '//ethn.io/62676.js';
```

Comment:
It looks like there's already a script store to load scripts in: https://github.com/ithaka/aiw-ui/blob/01265f0bf037d087a0f139abe227f33cb6b1e5a5/src/app/_services/script.store.ts#L6-L13 

And load it with `this._script.loadScript('ethnio-survey')`

Reply:
oh nice. didn't know about that.
updated to use this